src/core/HttpResponse.ts

```diff
@@ -34,10 +35,9 @@ interface StrictResponse<BodyType extends DefaultBodyType> extends Response {
  *
  * @see {@link https://mswjs.io/docs/api/http-response `HttpResponse` API reference}
  */
-export class HttpResponse<BodyType extends DefaultBodyType>
-  extends FetchResponse
-  implements StrictResponse<BodyType>
```

Comment:
With `StrictResponse` aliasing `HttpResponse`, it seems wrong to implement the type. While TypeScript does not complain about it, it would just express "`HttpResponse` implements `HttpResponse`". Seems unnecessary.

Reply:
Yeah, good catch. As long as HttpResponse has the same body symbol type, it should be okay to drop `implements`.

Reply:
Just to be sure, I marked the `bodyType` field on `HttpResponse` as `readonly` as well to match the previous interface field exactly.